web/elm/src/Dashboard/Grid/Constants.elm

```diff
@@ -21,17 +21,21 @@ cardBodyHeight =
 
 padding : number
 padding =
-    25
+    32
 
 
 cardHeaderHeight : number -> number
 cardHeaderHeight numRows =
-    20 + 30 * numRows
+    32 + 21 * numRows
+
+
+
+-- padding top 32 + line height 25 + spacing bottom 32
```

Comment:
Maybe we can encode these as variables and use them in `Dashboard/Styles.elm` so they can't get out of sync? Then we wouldn't need this comment

Reply:
![Screen Shot 2021-02-19 at 9 40 56 AM](https://user-images.githubusercontent.com/1585949/108518517-a2632700-7296-11eb-9bb9-7a00c3f21517.png)
![Screen Shot 2021-02-19 at 9 41 05 AM](https://user-images.githubusercontent.com/1585949/108518518-a2fbbd80-7296-11eb-90b7-16b53faa1e5b.png)

these are some samples when vertical spacing between header rows is set to 8px